src/accountingservice/genproto/oteldemo/demo.pb.go

```diff
@@ -15,7 +15,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
All go code in this PR is autogenerated, no manual change here. Also, non of the Go services relied on the GRPC interfaces that have been changed, so these services should not be affected.